app/src/views/EapRegistration/index.tsx

```diff
@@ -1,21 +1,259 @@
+import {
+    Container,
+    DateInput,
+    InputSection,
+    RadioInput,
+    SelectInput,
+    TextInput,
+} from '@ifrc-go/ui';
 import { useTranslation } from '@ifrc-go/ui/hooks';
+import {
+    numericIdSelector,
+    stringNameSelector,
+} from '@ifrc-go/ui/utils';
 
+import CountrySelectInput from '#components/domain/CountrySelectInput';
+import DisasterTypeSelectInput from '#components/domain/DisasterTypeSelectInput';
+import NationalSocietySelectInput from '#components/domain/NationalSocietySelectInput';
 import Page from '#components/Page';
 
 import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+interface Props {
+    disabled?: boolean;
+}
+
+const eapFormOptions = [
+    { value: 'full', label: 'Full application' },
+    { value: 'simplified', label: 'Simplified application' },
+    { value: 'not_sure', label: 'Not Sure' },
+];
 
 /** @knipignore */
 // eslint-disable-next-line import/prefer-default-export
-export function Component() {
+export function Component(props: Props) {
+    const {
+        disabled,
+    } = props;
+
     const strings = useTranslation(i18n);
 
     return (
         <Page
+            className={styles.eapRegistrationForm}
             heading={strings.eapRegistrationHeading}
             description={strings.eapRegistrationDescription}
+            withBackgroundColorInMainSection
         >
-            {/* TODO: Add the form */}
-            Application Details
+            <Container
+                heading="Application Details"
+                childrenContainerClassName={styles.content}
+            >
+                <InputSection
+                    title="National Society (NS)"
+                    description="Select National Society that is planning to apply for the EAP"
+                    withAsteriskOnTitle
+                >
+                    <NationalSocietySelectInput
+                        error={undefined}
+                        name="national_society"
```

Comment:
This is a good go ahead, but @barshathakuri , you need to create a typing, setup form and then create inputs. The form needs to be editable and working, only the send action is stopped. Only after we do this, we can change things easily later on.